.github/workflows/ci.yml

```diff
@@ -18,7 +18,6 @@ jobs:
       fail-fast: false
       matrix:
         versions:
-        - {os: 'ubuntu-20.04', otp: '21.x', rebar: '3.15.2'}
```

Comment:
I'm having to drop CI for 21.

Reply:
It should all still work, but a version of meck that works with 21 will not work with 27.